spec/services/step_by_step_filter/options_spec.rb

```diff
@@ -2,69 +2,17 @@
 
 RSpec.describe StepByStepFilter::Options do
   it "returns filter options with the default selected" do
-    subject = described_class.statuses
-
-    all_option = {
-      text: "All",
-      data_attributes: {
-        show: "all",
-      },
-      selected: true,
-    }
-
-    expect(subject).to include(all_option)
-
-    StepByStepPage::STATUSES.each do |status|
-      status_option = {
-        text: status.humanize,
-        value: status,
-        data_attributes: {
-          show: status,
-        },
-        selected: false,
-      }
-
-      expect(subject).to include(status_option)
-    end
+    expect(described_class.statuses)
+      .to include(text: "All",
+                  data_attributes: { show: "all" },
+                  selected: true)
   end
 
   it "returns filter options with the selected status" do
-    subject = described_class.statuses("scheduled")
-
-    all_option = {
-      text: "All",
-      data_attributes: {
-        show: "all",
-      },
-      selected: false,
-    }
-
-    expect(subject).to include(all_option)
-
-    scheduled_option = {
-      text: "Scheduled",
-      value: "scheduled",
-      data_attributes: {
-        show: "scheduled",
-      },
-      selected: true,
-    }
-
-    expect(subject).to include(scheduled_option)
-
-    other_options = StepByStepPage::STATUSES.dup - %w(scheduled)
-
-    other_options.each do |status|
-      status_option = {
-        text: status.humanize,
-        value: status,
-        data_attributes: {
-          show: status,
-        },
-        selected: false,
-      }
-
-      expect(subject).to include(status_option)
-    end
+    expect(described_class.statuses("scheduled"))
+      .to include(text: "Scheduled",
+                  value: "scheduled",
+                  data_attributes: { show: "scheduled" },
+                  selected: true)
```

Comment:
Same here - otherwise test will pass even if we only have "Scheduled" in the filter list.